zeebe/clients/java/src/main/java/io/camunda/zeebe/client/api/command/TopologyRequestStep1.java

```diff
@@ -17,6 +17,7 @@
 
 import io.camunda.zeebe.client.api.response.Topology;
 
-public interface TopologyRequestStep1 extends FinalCommandStep<Topology> {
+public interface TopologyRequestStep1
+    extends CommandWithCommunicationApiStep<TopologyRequestStep1>, FinalCommandStep<Topology> {
```

Comment:
:thought_balloon: These changes might not fit in this PR, but I wanted to make the usage of the `CommandWithCommunicationApiStep` interface transparent with this PR. I can move it in a later commit if necessary.

Reply:
Fine with me, can stay 👍 🙂